src/components/ClearAllTags.tsx

```diff
@@ -4,10 +4,11 @@ const ClearAllTags = (props: {
   classNames: {
     clearAll: string,
   },
+  accessibilityLabel: string,
   onClick: () => void,
 }) => {
   return (
-    <button className={props.classNames.clearAll} onClick={props.onClick}>
+    <button aria-label={props.accessibilityLabel} className={props.classNames.clearAll} onClick={props.onClick}>
```

Comment:
I think we don't need a prop for this, and instead just add a descriptive aria label instead. If the user wants to override it later (if we get any such request) then we can consider supporting customization. But for now adding a hard-coded aria label should be good to go.
Let me know what you think

Reply:
Your comment makes sense, but a hard-coded aria label gives the information to assistive technology users that, on clicking CTA we are going to perform some task. But they will not have context what tags are these? What are they clearing all that stuff, even if we give a good descriptive hardcoded value 

They feel something like this  https://inaccessible.gallery/form-labels/

There is a saying 
 <blockquote>Accessibility isn't optional: It's integral to website performance and user experience</blockquote>

 To have a better accessible CTA, I feel we should consider taking the aria label from the user.

Reply:
The aria label - **"Clear all tags"** clearly conveys that it would be clearing all the tags. It doesn't clear some specific tags (which you can select to remove) so there isn't any extra info needed to convey about which specific tags will be removed since all tags will be removed at once. Hence I think an hardcoded label will be best suitable for this.

Still if you think the hardcoded label doesn't help, please elaborate with an example

Reply:
Below are the cases where we need to consider adding a description to clear all tags:

1. multiple tags on a single page
2.  when Voice over user passes through this CTA, they just hear as Clear all . I have replicated what they feel here  https://codepen.io/shrilakshmishastry/pen/PoMWzwv

If you still feel like hardcoded value is enough I will add it

Reply:
Do you mean when you have multiple React Tags Component on a single page ? In that case, I agree the hard-coded aria label wouldn't be intuitive and we should provide an option to pass the aria label.

Even if we allow customizing the aria label via prop, we will need to come up with a better config for aria attributes instead of just adding prop for every element as the props list will keep growing. 

Let me think over this on how better we can pass aria props and get back to you. If you have any suggestions do share.

Reply:
one way we can adopt is the below structure 
```
accessibilityProps?:{
labelOfclearAllCta?: string;
TagsLabelId?:string;
inputLabelId?:string;
}
```

Reply:
let me know what you think.

Reply:
let's keep it short and simple. We can tweak before release if needed but I would like to unblock you for now

```
ariaAttrs?: {
  clearAllLabel?: string
}
```

Reply:
lgtm. 
can we change clearAllLabel to clearAllButtonLabel to tell the label is for the button?

Reply:
I don't think its needed as its very intuitive and clear its for the the clear all button since there isn't any other element with the same functionality/label. So lets keep the prop short and simple

Reply:
ok I will update the prop name and push the changes 👍

Reply:
added above mentioned prop https://github.com/react-tags/react-tags/blob/b8590efe6ecfa94e8ea6d4b59b8666b0596a5b69/src/components/ReactTags.tsx#L44